src/components/Button/styles.module.scss

```diff
@@ -0,0 +1,44 @@
+@import 'scss/variables/_colors';
+
+.button {
+  border-radius: 5px;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 4px 12px;
+  width: 100%;
+}
+
+.contained {
+  composes: button;
```

Comment:
This caught my attention. Why choosing `composes` over for example sass nesting?

Reply:
I tought I was taking advantage of composes (and I hate nesting since working with Clemente haha), in this case I think is the same, what would be the wolox way?

Reply:
Haha, I believe that currently there is no opinion about the "composes" rule.
I asked because in my opinion it's more of a matter about the way you want to structure the project's scss classes and that the chosen structure works for you and the projects needs.